website/search_migration/__init__.py

```diff
@@ -277,36 +272,7 @@
       SELECT FALSE AS PENDING
     ) END)
             ) REGISTRATION_APPROVAL ON TRUE
-  LEFT JOIN LATERAL (
```

Comment:
Nodes are no longer indexed as preprints, so the search_migration should not be looking for preprint things on nodes.